Quake/gl_texmgr.c

```diff
@@ -566,6 +566,33 @@ void TexMgr_LoadPalette (void)
 	((byte *)&d_8to24table_conchars[0])[3] = 0;
 }
 
+/*
+================
+TexMgr_NearestColor
```

Comment:
This doesn't belong here. Has nothing to do with texture management.

Reply:
Agreed -- what would be a better spot for this? gl_draw.c maybe?

Reply:
Or maybe common.c?

Reply:
Are there plans to allow for changing colors? This seems to be only ever called with the pair of values (20, 20, 0), (40, 30, 15), so you could assume the default Quake palette is used and do `bgColor = 51; fillColor = 19;`. Otherwise  `gl_draw.c` may be the best place.